app/controllers/organisations_controller.rb

```diff
@@ -2,14 +2,12 @@ class OrganisationsController < ApplicationController
   before_action :set_organisation, only: %i[edit update]
   before_action :validate_user_is_part_of_organisation, only: %i[edit update]
 
-  def index; end
-
   def edit; end
 
   def update
     if @organisation.update(organisation_params)
-      redirect_to organisation_path(@organisation)
-      flash[:notice] = 'Organisation updated'
+      redirect_to team_members_path
```

Comment:
I was tempted the first time I saw this to create a new controller which isn't tied to team_members or organisations.

It could just be called team and host the team_members and organisation functionality.

Not vital, happy to discuss or not do this at all.  Just thought I'd mention it.